### DISCUSSION TITLE
Got it installed.. Not sure how to actually use this

### DISCUSSION THREAD

**Discussion by chiptesters** [sebr/bhyve-home-assistant#60] [General]
Thanks for making this!  I pasted the following into my configuration.YAML:

bhyve:
  username: !secret bhyve_username
  password: !secret bhyve_password

sensor:
  - platform: bhyve

switch:
  - platform: bhyve

And I'm getting an error:  "Secret bhyve_username not defined".  I also tried actually putting my username and password in these two sports and I'm still getting an error.

**Comment by chiptesters** [comment]
Okay so I just wrote my username and password and got rid of the entire "!secret XXXX" text.  That seems to have passed the configuration test, but I'm still not seeing any entities or cards I can add related to B-Hyve.

**Comment by sebr** [comment]
1. Please refer to the Home Assisrant documentation: [storing secrets](https://www.home-assistant.io/docs/configuration/secrets/)
2. This integration does not provide any Lovelace cards for you to add to your home assistant dashboards. You must build your desired layout using the entities which this integration makes available.
3. If you are not seeing any entities please raise a separate issue and fill out the requested information. Please ensure your b-hyve device is connected to the internet using a wifi bridge. Bluetooth is not supported.

**Reply by chiptesters** [reply]
Thanks Sebr for the reply!  I'll do all three of those and get back to you.  Indeed I don't have the wifi bridge, I will procure one of those as well.
